### ISSUE TITLE
What have you done?

### ISSUE THREAD

**Issue by 38github** [GuitarML/SmartGuitarPedal#11]
This sounds so much better compared to a year ago. I am impressed! SO IMPRESSED!

I just sent you a small donation and I hope that covers the coffee for a while. I really hope more people will help out getting you/this software attention and support.

**Comment by GuitarML** [comment]
@38github Hey thanks so much! It's really just from understanding how these neural networks operate better, the Wavenet model seems to sound the best in most cases, although cpu usage is still high.  I changed the default hyperparameters to use more layers with less channels, and that seems to sound better while dropping cpu usage slightly lower.  Also I normalize the data by default now, which improves accuracy and keeps volume consistent.  More improvements and more models coming so check back for updates!
